pcsx2/CDVD/CsoFileReader.cpp

```diff
@@ -52,7 +53,7 @@ CsoFileReader::~CsoFileReader()
 bool CsoFileReader::CanHandle(const std::string& fileName, const std::string& displayName)
 {
 	bool supported = false;
-	if (StringUtil::EndsWith(displayName, ".cso"))
+	if (StringUtil::EndsWith(displayName, ".cso") || StringUtil::EndsWith(displayName, ".zso"))
```

Comment:
```suggestion
	if (displayName.ends_with(".cso") || displayName.ends_with(".zso"))
```

Been meaning to do a pass through to replace these with the C++20 equivalents, but since you're editing this line, may as well do this one here.

Reply:
But then again, we use fmt::format instead of std::format, I agree that we should use C++20 more often, but please be consistent. You can't expect me to use it sometimes yes sometimes no, it gets confusing.

Reply:
Please read what I said again - I stated that we will be going through and removing the helper function, but since you're changing this line anyway, you may as well update it.

Reply:
Because std::format is not available in the libstdc++ versions we use on Ubuntu, nor the libc++ version used on MacOS.

Reply:
Oh damn that's a bummer. Alright I'll do the changes you've asked. Shouldn't take much.